io.github.itprojects.MasVisGtk.json

```diff
@@ -0,0 +1,102 @@
+{
+    "id" : "io.github.itprojects.MasVisGtk",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "46",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "masvisgtk",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=home"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name":"openblas",
+            "no-autogen":true,
+            "make-args":[
+                "DYNAMIC_ARCH=1",
+                "FC=gfortran",
+                "NO_CBLAS=1",
+                "NO_LAPACKE=1",
+                "USE_OPENMP=0"
+            ],
+            "make-install-args": [
+                "PREFIX=/app"
+            ],
+            "sources":[
+                {
+                "type":"archive",
+                "url":"https://github.com/OpenMathLib/OpenBLAS/archive/refs/tags/v0.3.28.tar.gz",
+                "sha256":"f1003466ad074e9b0c8d421a204121100b0751c96fc6fcf3d1456bd12f8a00a1"
+                }
+            ]
+        },
+        {
+            "name":"lapack",
+            "buildsystem":"cmake-ninja",
+            "builddir":true,
+            "cleanup":[
+                "/lib/cmake"
+            ],
+            "config-opts":[
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
+                "-DBUILD_SHARED_LIBS=ON",
+                "-DBUILD_TESTING=OFF",
+                "-DLAPACKE=ON",
+                "-DCBLAS=ON"
+            ],
+            "sources":[
+                {
+                    "type":"archive",
+                    "url":"https://github.com/Reference-LAPACK/lapack/archive/refs/tags/v3.12.0.tar.gz",
+                    "sha256":"eac9570f8e0ad6f30ce4b963f4f033f0f643e7c3912fc9ee6cd99120675ad48b"
+                }
+            ]
+        },
+        {
+            "name": "qhull",
+            "buildsystem": "cmake-ninja",
+            "config-opts":[
+                "-DBUILD_STATIC_LIBS=OFF"
+            ],
+            "sources": [
+                {
+                "type": "archive",
+                "url": "https://github.com/qhull/qhull/archive/refs/tags/2020.2.tar.gz",
+                "sha256": "59356b229b768e6e2b09a701448bfa222c37b797a84f87f864f97462d8dbc7c5"
+                }
+            ],
+            "cleanup":[
+                "/bin",
+                "/share/man",
+                "/share/doc"
+            ]
+        },
+        "python3-deps.json",
+        {
+            "name" : "masvisgtk",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/itprojects/MasVisGtk.git",
+                    "tag": "v3.0.0"
```

Comment:
add the commit that matches the tag here. this protects against retagging.

Reply:
What would happen on the server if the tag is re-tagged?

Reply:
you shouldn't be retagging - it's a bad practice (and nothing to do with flatpak), so yeah the build will fail and that's the intent.

Reply:
Yet, if there is no first/initial flathub repository for this app, then that would be acceptable for testing purposes?

Reply:
then use a commit and tag when it's ready.

Reply:
again this has nothing to do with flatpak. It's a bad practice overall. A published tag SHOULD be IMMUTABLE.

Reply:
Added a commit.